pkg/scheduler/scheduler.go

```diff
@@ -203,7 +207,9 @@ func (s *Scheduler) assign(ctx context.Context, e *entry) {
 	go func() {
 		err := s.client.Update(ctx, newWorkload)
 		if err == nil {
-			log.V(2).Info("Successfully assigned capacity and resource flavors to workload")
+			msg := fmt.Sprintf("Successfully assigned %v to capacity %v", workload.Key(newWorkload), e.Capacity)
+			s.recorder.Eventf(newWorkload, corev1.EventTypeNormal, "Scheduled", msg)
```

Comment:
should we avoid the term "scheduled" especially in user-facing logs to avoid confusion with kube-scheduler meaning of scheduled? may be use "Assigned" here.

Reply:
SG
Done